changelog/fix-TEC-5410-hide-install-ET-button

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Added CSS that will hide the Install Event Tickets button following completing the Onboarding Wizard. [TEC_5410]
```

Comment:
The ticket reference uses an underscore instead of a dash. 😝 
Also, does it hide the button when you complete the wizard? Or does it hide the button when you have ET installed? There's a difference.

Reply:
The correct behavior fix would be "it hides it when you complete step 5 of the wizard" as the button is not rendered if ET is installed prior to running the wizard.

That's what the new styles are doing - the wizard is the one that marks the item via the `tec-admin-page__onboarding-step--completed` class

Reply:
```suggestion
Added CSS that will hide the Install Event Tickets button following completing the Onboarding Wizard. [TEC-5410]
```